iceoryx_hoofs/test/moduletests/test_cxx_helplets.cpp

```diff
@@ -143,61 +143,72 @@ TEST_F(Helplets_test, maxAlignment)
     EXPECT_THAT((iox::cxx::maxAlignment<FooBar, FuBar>()), Eq(alignof(FooBar)));
 }
 
-TEST_F(Helplets_test, bestFittingTypeUsesUint8WhenValueSmaller256)
+TEST_F(Helplets_test, ArrayCapacityReturnsCorrectValues)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "8392b2ba-04ef-45e6-8b47-4c0c90d98f61");
+    constexpr uint64_t CAPACITY{42};
+    // NOLINTJUSTIFICATION Used only for test purposes
+    // NOLINTNEXTLINE(cppcoreguidelines-avoid-c-arrays, hicpp-avoid-c-arrays)
+    constexpr uint32_t SUT[CAPACITY]{};
+
+    EXPECT_EQ(iox::cxx::arrayCapacity(SUT), CAPACITY);
+}
+
+TEST_F(Helplets_test, BestFittingTypeUsesUint8WhenValueSmaller256)
```

Comment:
Somehow I think these tests could all be one test that `BestFittingType` is the appropriate type (it is all compile time and there is no interference between tests). But never mind, it was like this before.